src/agents/gc/README.md

```diff
@@ -53,4 +53,146 @@ sudo pwsh
 $x = Get-GuestConfigurationPackageComplianceStatus -path <path to the ZIP>/OSConfig_Proto_Policy.zip -Verbose
 $x.resources[0].properties
 ```
-In addition to the MC traces written to the the PowerShell console, you can also see the NRP's own log at `/var/log/osconfig_gc_nrp.log`.
\ No newline at end of file
+In addition to the MC traces written to the the PowerShell console, you can also see the NRP's own log at `/var/log/osconfig_gc_nrp.log`.
+
+## 5. Onboarding a device to Arc
+
+Go to Azure Portal | Azure Arc | Add your infrastructure for free | Add your existing server | Generate script. Specify the desired Azure subscription, resource group and OS (Linux) for the device. Copy and run the generated instructions in the local shell on the device to be onboarded.

```

Comment:
Instructions are clear, but may be easier to read as a list? For example:

1. Go to Azure Portal
2. Select Azure Arc
3. ...